backend/src/apiserver/template/tekton_template.go

```diff
@@ -299,7 +299,7 @@ func (t *Tekton) injectArchivalStep(workflow util.Workflow, artifactItemsJSON ma
 					// move all results to /tekton/home/tep-results to avoid result duplication in copy-artifacts step
 					// TODO: disable eof strip, since no results under /tekton/results after this step
 					moveResults := workflowapi.Step{
-						Image:   "busybox",
+						Image:   "quay.io/rmartine/busybox",
```

Comment:
Is there a shared busybox location in quay that we can refer to instead of one under your personal profile? Barring that, is there a kfp-tekton/opendatahub location we could push to?

Reply:
Actually, this shouldn't be part of the code since this was only to make it work for my environment. I'll revert this change.